config/locales/en.yml

```diff
@@ -885,6 +886,45 @@ en:
+        pagepile: PagePile
+        pagepile_short_desc: PagePile is a tool that allows you to create a list of pages which doesn't change over time and is identified by a unique ID
+        templates_desc: >
+          A template is a Wikipedia page created to be included in other pages. Templates usually contain repetitive material that might need to show up on a larger number of articles or pages. They are commonly used for boilerplate messages, standardized warnings or notices, infoboxes, navigational boxes, and similar purposes.
+        pagepile_desc: >
+          PagePile is a tool developed by the Wikimedia Foundation for the Wikimedia projects, including Wikipedia. It allows users to collect and organize a group of web page links into a single bundle called a "pile." These piles can then be easily shared with others, enabling collaborative work and information sharing.
+
+          PagePile serves as a convenient way to gather and present a curated set of Wikipedia articles or other web pages related to a specific topic or theme. It can be used for various purposes, such as creating reading lists, compiling research materials, organizing articles for a project, or simply bookmarking a collection of pages for personal reference
+        petscan_desc: >
+          PetScan is another tool developed by the Wikimedia Foundation that allows users to generate custom lists of Wikipedia pages based on specific criteria. It is a powerful tool for retrieving and filtering information from Wikipedia's vast database of articles.
+
+          PetScan offers a user-friendly interface that allows users to specify various parameters to create highly targeted lists. These parameters include categories, namespaces, templates, linked pages, page properties, and more. Users can combine multiple criteria to refine their search and generate a customized list of articles that meet their specific requirements.
+        petscan_have_psid_title: I have a PetScan ID
+        petscan_not_have_psid_title: I don't have a PetScan ID
+        petscan_have_psid_desc: >
+          Use this if you have already built a PetScan query and have the ID for it. You can find the ID in the URL of the PetScan query. For example, if the URL of the PetScan query is https://petscan.wmflabs.org/?psid=1234567, then the ID is 1234567.
+
+          If you don't have a PetScan ID, you can build a query using the PetScan tool and then come back here to enter the ID.
+        petscan_not_have_psid_desc: >
+          Use this if you don't have a PetScan ID and wish to use a simplified builder to create it. This lacks some of the advanced features of the PetScan tool, but is easier to use and you don't have to leave the dashboard to build the query!
+        petscan_create_psid: Create a PetScan ID using the official PetScan tool
```

Comment:
What is this used for?

Reply:
![image](https://github.com/WikiEducationFoundation/WikiEduDashboard/assets/10794178/663529b2-2c90-403a-b585-8b1650e01b76)

in here

Reply:
I don't see "Create a PetScan ID using the official PetScan tool" anywhere in that screenshot.

Reply:
Ah you were referring to that - its here 
![image](https://github.com/WikiEducationFoundation/WikiEduDashboard/assets/10794178/d32f677a-f5ee-4ae2-9abb-4e1304f8ba2b)